python/instrumentation/openinference-instrumentation-llama-index/pyproject.toml

```
@@ -28,7 +28,7 @@ dependencies = [
   "opentelemetry-api",
   "opentelemetry-instrumentation",
   "opentelemetry-semantic-conventions",
-  "openinference-instrumentation>=0.1.7",
+  "openinference-instrumentation>=0.1.9",
   "openinference-semantic-conventions",
```

not sure why this didn't matter
```suggestion
  "openinference-semantic-conventions>=0.1.9",
```

---

Because the instrumentation requires it, so it's implicit